packages/manager/modules/pci/src/projects/project/storages/databases/database/users/modify-password/modify-password.html

```diff
@@ -0,0 +1,29 @@
+<form name="modifyPassword">
+    <oui-modal
+        data-heading="{{:: 'pci_databases_users_modify_password_title' | translate }}"
+        data-primary-label="{{:: 'pci_databases_users_modify_password_confirm' | translate }}"
+        data-primary-action="$ctrl.modifyPassword()"
+        data-primary-disabled="!modifyPassword.$valid"
+        data-secondary-label="{{:: 'pci_databases_users_modify_password_cancel' | translate }}"
+        data-secondary-action="$ctrl.goBack()"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isLoading"
+    >
+        <oui-field
+            data-label="{{:: 'pci_databases_users_modify_password_label' | translate }}"
+            data-help-text="{{:: 'pci_databases_users_modify_password_help_text' | translate }}"
+            data-size="xl"
+        >
+            <input
+                class="oui-input"
```

Comment:
Actually, we only have the re-generate password in the specs. Since the PU wanted the user to be able to provide the password, we added the password field. I can check on this...